Library/Formula/cvc4.rb

```diff
@@ -0,0 +1,40 @@
+class Cvc4 < Formula
+  homepage "http://cvc4.cs.nyu.edu/"
+  url "http://cvc4.cs.nyu.edu/builds/src/cvc4-1.4.tar.gz"
+
+  devel do
+    url "http://cvc4.cs.nyu.edu/builds/src/unstable/latest-unstable.tar.gz"
```

Comment:
cvc4 only offers dates for their development releases: http://cvc4.cs.nyu.edu/builds/src/unstable/

Is there a way around this, like having a templated URL that would search for the most recent date? I went with `1.5-prerelease` as a manual version, since that's what they use in the project.